yaml_files/simphony_metadata.yml

```diff
@@ -436,6 +484,50 @@ CUDS_KEYS:
     definition: Periodic boundary condition (PBC)
     models: [CUBA.ELECTRONIC, CUBA.ATOMISTIC, CUBA.MESOSCOPIC, CUBA.CONTINUUM]
 
+  SHEAR_STRESS_POWER_LAW_SLIP_VELOCITY:
+    parent: CUBA.CONDITION
+    definition: Shear stress power law dependant slip velocity boundary condition
+    models: [CUBA.CONTINUUM]
+    CUBA.VARIABLE:
```

Comment:
Each condition can be applied on one or more field variables (in general on the whole domain or specifically on boundaries). So each condition has a CUBA.VARIABLE property which is a list of variables that the condition applies to. These variables themselves should be defined in the CUBA set (currently at least in cuba.yml). We could have called it AFFECTED_VARIABLE, but this is unnecessary as it is clear from the context to intended users and will just increase the amount of CUBA without real need. However, since in this particular condition, the velocity is the only affected variable, one could set it on the metadata level by using lower case: variable: CUBA.VELOCITY (however, we should be clear whether this needs to apply to one particular phase?)